ember_debug/general-debug.js

```diff
@@ -19,17 +19,18 @@ const GeneralDebug = EmberObject.extend(PortMixin, {
   // Keep an eye on https://github.com/ember-cli/ember-cli/issues/3045
   emberCliConfig: computed(function() {
     let config;
-    $('meta[name]').each(function() {
-      const meta = $(this);
-      let match = meta.attr('name').match(/environment$/);
+    let metas = document.querySelectorAll('meta[name]');
+    for (let i = 0; i < metas.length; i++) {
```

Comment:
Last time I checked this kind of low-lever optimizations had a despicable impact on performance, but feel free to create a PR to improve it.